pages/data/manifest.json

```diff
@@ -42,11 +42,17 @@
     "xata": "Xata"
   },
   "providers": {
-    "42-school": "42 School",
+    "github": "GitHub",
+    "google": "Google",
+    "twitter": "Twitter",
     "apple": "Apple",
+    "discord": "Discord",
+    "auth0": "Auth0",
+    "resend": "Resend",
+    "sendgrid": "Sendgrid",
+    "42-school": "42 School",
```

Comment:
I would keep the email providers separate from the OAuth ones 💅🏽 

For email, I guess a database does need to be set right ( _to store verification tokens..._ ), so the instructions can't be the same as OAuth and I wouldn't group them together.

Suggest we rename this object to:
```json
"providers-oauth": {
  "github": "Github",
   ...
 }, 
 "providers-email": {
  "resend": "Resend",
   ...
 }, 
 ```

Reply:
Yeah good idea. Pushed!